android-components/components/support/base/src/main/java/mozilla/components/support/base/dialog/DeniedPermissionDialogFragment.kt

```diff
@@ -36,7 +37,7 @@ class DeniedPermissionDialogFragment : DialogFragment() {
             .setPositiveButton(R.string.mozac_support_base_permissions_needed_positive_button) { _, _ ->
                 openSettingsPage()
             }
-        return builder.create()
+        return builder.create().withCenterAlignedButtons()
```

Comment:
We could do two things here:
1. Copy over the extension functions to this component to avoid making ui-widget a dependent.
2. Move this `DeniedPermissionDialogFragment` somewhere else. It seems suspicious that it's here. cc @Amejia481 - would you know a better place to move this fragment too?

Reply:
Yeah, it's really odd that it was part of the `support-base`,  @mavduevskiy already moved it to feature download that seems a better fit.